sovrin_client/client/client.py

```diff
@@ -121,7 +121,7 @@ def postReplyRecvd(self, identifier, reqId, frm, result, numReplies):
                     # a malformed message should not result in an error message
                     # being shown on the cli since the clients would anyway
                     # collect enough replies from other nodes.
-                    logger.error("Observer threw an exception", exc_info=ex)
```

Comment:
@khagesh , why cli was showing "Observer threw an exception", did we fix the root cause as well. And since it was logged as error earlier, we came to know about it. But now, if this will be debug, how we'll know if something is going wrong behind the scene?